### ISSUE TITLE
lighter library

### ISSUE THREAD

**Issue by muscaiu** [hannseman/homebridge-mi-hygrothermograph#84]
Is there any chance you will make a lighter library to be used without things like Homebridge?
I'm sure other people like me are interested in using it to read the Mi sensor on a simple Express Node server or something similar.

**Comment by hannseman** [comment]
Yes this is something that I’ve been meaning to do. I’ll look into publishing a new package ASAP.

**Comment by Hypfer** [comment]
@hannseman you might want to take a look at https://github.com/Hypfer/Cybele

It should hopefully probably maybe provide a framework to do so. Utilizing BlueZ via DBUS to completely ditch that broken noble module

Feedback/PRs much appreciated

**Comment by hannseman** [comment]
@Hypfer that's a very interesting approach. It would unfortunately require dropping support for non-linux systems. 

Since there's so many issues with the noble module (almost all open issues are noble / node-bluetooth-hci-socket related) I actually might consider moving to use bluez over dbus instead. One could imagine a homebridge-mi-hygrothermograph 3.0 which uses dbus and windows / MacOS users could stay on 2.0 and receive patches for breaking issues. 🤔

**Comment by alan-strohm** [comment]
I'd like to reuse the parser for an [integration](https://github.com/mKeRix/room-assistant/issues/176) with room-assistant.  The parser has very few dependencies (i.e. it could be used with either noble or Cybele) so it would be an ideal candidate for a separate package.  Is that something you'd be interested in doing, @hannseman ?

**Comment by muscaiu** [comment]
I used to have a way (library) to scan for bluetooth device id's in range like a year ago (i didn't document it....)
What are you guys using to detect the Mi sensors mac or id?
